src/D2L.Security.OAuth2/Scopes/Scope.cs

```diff
@@ -124,39 +124,38 @@ public Scope( string group, string resource, string permission )
 		/// *:*:read                    => Read only for all resources in all groups
 		/// </example>
 		public static bool TryParse( string scopePattern, out Scope scope ) {
+			try {
+				scope = Parse( scopePattern );
+				return true;
+			} catch {

```

Comment:
:(  Not a blocker, but I don't love it

Reply:
What would you rather? Just catch ArgumentException?

Reply:
No, I just hate exceptions.  It's fine if this is all mostly constants